README.md

```diff
@@ -156,15 +156,35 @@ git clone git@github.com:OpenAssetIO/OpenAssetIO
 Linux is currently the only fully tested platform. Support
 for other platforms is the subject of ongoing work.
 
+#### Quick start: Python only
+
+For use in a Python-only environment, the Python package can be
+installed from the root of the repository using
+
+```shell
+pip install .
+```
+
+This will automatically build and install the Python extension module,
+along with the Python sources.
+
+Note that CMake's `find_package` must be able to locate dependencies.
+Consider using the [`CMAKE_TOOLCHAIN_FILE`](https://cmake.org/cmake/help/v3.24/envvar/CMAKE_TOOLCHAIN_FILE.html)
+environment variable to augment CMake's package search paths.
+
```

Comment:
[consider] Given we're going this far, until we sort #624, is it worth just putting the few lines of `conan install` here instead of more general toolchain details? This has been unfamiliar turf for many attempting to get going with python only builds.

Reply:
Added some additional instructions in a2cf809f, how does that look?

Reply:
I'd argue that the Conan install should be executed from the setup.py if you want to make a good user experience and will anyway be required for sdists.

Reply:
We're going to look into this, just in a separate task #666

Reply:
@JeanChristopheMorinPerso Do you have any experiencing in making this work on Windows too - is it fair to expect that people are already in a developer command prompt?

Reply:
Yep, it is entirely fair to expect that the user has a compiler available and that he is in a developer command prompt properly setup to compile C++ code. You do that same assumption on Linux and macOs, so it would be weird to make a different assumption on Windows.